### DISCUSSION TITLE
Error: taxonomy, diamond

### DISCUSSION THREAD

**Discussion by joseailtoncruz** [bbuchfink/diamond#638] [General]
I am using diamond, when executing the command line <**./diamond blastx -d reference.dmnd -q queries.fasta -o teste --outfmt 5> I can run the program normally, but I would like to obtain the taxonomy, follow the instructions of the manual but an error message always appears (diamond v0.9.14.115 | by Benjamin Buchfink <buchfink@gmail.com> Licensed under the GNU AGPL <https://www.gnu.org/licenses/agpl.txt**> Check http://github.com/bbuchfink/diamond for updates.
#CPU threads: 12
Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
#Target sequences to report alignments for: 25
Temporary directory:
Opening the database... [0.000482s]
Loading taxonomy... No such file or directory
  [0.000119s]
**Error: Error opening file prot.accession2taxid.FULL.gz**)
the command line I used was the following <**./diamond blastx -d reference.dmnd -q queries.fasta -o matches --taxonmap prot.accession2taxid.FULL.gz --outfmt 5**>, the file used in --taxonmap was downloaded from ncbi (https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/), that is, following the instructions "**Path to the output DIAMOND database file.

--taxonmap <file>

Path to mapping file that maps NCBI protein accession numbers to taxon ids (gzip compressed). This parameter is optional and needs to be supplied in order to provide taxonomy features. The file can be downloaded from NCBI: ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/accession2taxid/prot.accession2taxid.FULL.gz

Versions older than v2.0.7 only support the reduced mapping file: ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/accession2taxid/prot.accession2taxid.gz

A custom file following the same format may be supplied here. Note that the first line of this file is assumed to contain headings and will be ignored.**". I would like to know how I can obtain the taxonomy of the isolates I am analyzing.

Thank you in advance for your help, and I apologize for my English.

**Comment by bbuchfink** [comment]
Try to use the latest version of diamond and use the `--taxonmap` option as part of the `makedb` call.
